srcts/src/main.ts

```diff
@@ -4061,6 +4062,17 @@ function main(): void {
           $(sheet.ownerNode).remove();
         };
 
+        // There doesn't appear to be proper way to wait until new styles have been
+        // _applied everywhere_ so we repeatedly call bindAll() every .1s for 10 seconds.
+        // (bindAll() is used as an indirect way of calling doSendTheme() which resends CSS
+        // info stored in input values accessible to shiny::getCurrentOutputInfo()).
+        /* global Shiny */
+        let scheduleCssReporter = function () {
+          let handle = setInterval(Shiny.bindAll, 100);
```

Comment:
Instead of `Shiny.bindAll` lets just send the CSS info (`sendImageSize`) and throttle calls to it.